src/A2A/JsonRpc/JsonRpcResponse.cs

```diff
@@ -14,8 +14,13 @@ public class JsonRpcResponse
     public string Id { get; set; } = string.Empty;
 
     [JsonPropertyName("result")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
As long as serialization is being done with A2AJsonUtilities, this is not necessary, as that shared options instance has this set:
https://github.com/a2aproject/a2a-dotnet/blob/d2b7d7948ef6b0d51f7ce6be720d7a63f0b31031/src/A2A/A2AJsonUtilities.cs#L31